File: hospitalManagement/Neurology.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace hospitalManagement
{
    internal class Neurology:Faculty
    {
        //Field
        // Infor field
        //  Management field
        // Dynamic field
        // Properties
        // Constructors
        public Neurology()
        {

        }

        public Neurology(string id, string name, string description, string room, Employees employeeList, Patients patientList) : base(id, name, description, room, employeeList, patientList)
        {
        }
        // Destructors
        ~ Neurology()
        {

        }
        // Methods
        // in, output
        public override void Input()
        {
            base.Input();
        }

        public override void Output()
        {
            base.Output();
        }
        // General method
        // Other method
        // Overriding
    }
}

File: src/hospitalManagement/Neurology.java
package hospitalManagement;

import java.util.Scanner;
public class Neurology extends Faculty
    {
        //Field
        // Information field
        //  Management field
        static Scanner input = new Scanner(System.in);
        // Dynamic field
        // Properties
        // Constructors
        public Neurology()
        {

        }

        public Neurology(String id, String name, String description, String room, Doctors doctorList, Patients patientList)
        {
            super(id, name, description, room, doctorList, patientList);
        }
        
        // Methods
        // in, output
        public void Input()
        {
            super.Input();
        }

        public void Input1()
        {
            super.Input1();
        }

        public void Output()
        {
            super.Output();
        }
        // General method
        // Other method
        // Overriding
    }